README.md

```diff
@@ -1,11 +1,203 @@
+
+# Then read the prophecy to confirm it was created with the claim added
+ebcli query ethbridge get-prophecy 0 0x7B95B6EC7EbD73572298cEf32Bb54FA408207359 --trust-node
+
+# And finally, confirm that the prophecy was successfully processed and that new eth was minted to the testuser address
+ebcli query account $(ebcli keys show testuser -a) --trust-node
+
+```
+
+## Using the application from rest-server
+
+First, run the cli rest-server
+
+```
+ebcli rest-server --trust-node
+```
+
+An api collection for Postman (https://www.getpostman.com/) is provided [here](./cosmos-ethereum-bridge.postman_collection.json) which documents some API endpoints and can be used to interact with it.
+Note: For checking account details/balance, you will need to change the cosmos addresses in the URLs, params and body to match the addresses you generated that you want to check.
+
+## Running the relayer service
+
+For automated relaying, there is a relayer service that can be run that will automatically watch and relay events.
+
+```
+# Check ebrelayer connection to ebd
+ebrelayer status
+
+# Initialize the Relayer service for automatic claim processing
+ebrelayer init testing wss://ropsten.infura.io/ws 3de4ef81Ba6243A60B0a32d3BCeD4173b6EA02bb "LogLock(bytes32,address,bytes,address,uint256,uint256)" validator
+
+# Enter password and press enter
+# You should see a message like:  Started ethereum websocket... and Subscribed to contract events...
+```
+
+The relayer will now watch the contract on Ropsten and create a claim whenever it detects a lock event.
+
+## Using the bridge
+
+With the application set up and the relayer running, you can now use Peggy by sending a lock transaction to the smart contract. You can do this from any Ethereum wallet/client that supports smart contract transactions.
+
+The easiest way to do this for now, assuming you have Metamask setup for Ropsten in the browser is to use remix or mycrypto as the frontend, for example:
+ - 1. Go to remix.ethereum.org
+ - 2. Compile Peggy.sol with solc v0.5.0
+ - 3. Set the environment as Injected Web3 Ropsten
+ - 4. On 'Run' tab, select Peggy and enter "0x3de4ef81Ba6243A60B0a32d3BCeD4173b6EA02bb" in 'At Address' field
+ - 5. Select 'At Address' to load the deployed contract
+ - 6. Enter the following for the variables under function lock():
+  _recipient = [HASHED_COSMOS_RECIPIENT_ADDRESS] *(for testuser cosmos1pjtgu0vau2m52nrykdpztrt887aykue0hq7dfh, enter "0x636f736d6f7331706a74677530766175326d35326e72796b64707a74727438383761796b756530687137646668")*
+  _token = [DEPLOYED_TOKEN_ADDRESS] *(erc20 not currently supported, enter "0x0000000000000000000000000000000000000000" for ethereum)*
+  _amount = [WEI_AMOUNT]
+ - 7. Enter the same number from _amount as the transaction's value (in wei)
+ - 8. Select "transact" to send the lock() transaction
+
+## Using the modules in other projects
```

Comment:
Hmm, looks like this could be a Github bug, rendering italics in the markdown code and not detecting the end of the italics?

The italics are only restricted to the stuff in brackets when rendered by an actual Markdown renderer, say when looking at the actual README on the repo page: https://github.com/musnit/peggy/tree/unidirectional-complete

Or maybe I'm missing something lol?

Reply:
the solution should be that this block should be wrapped in ` as it starts with an underscore (i.e italics):
  
_recipient = [HASHED_COSMOS_RECIPIENT_ADDRESS] // (for test user cosmos1pjtgu0vau2m52nrykdpztrt887aykue0hq7dfh, enter "0x636f736d6f7331706a74677530766175326d35326e72796b64707a74727438383761796b756530687137646668")
  _token = [DEPLOYED_TOKEN_ADDRESS] // erc20 not currently supported, enter "0x0000000000000000000000000000000000000000" for ethereum)
  _amount = [WEI_AMOUNT]

to 


```solidity
  _recipient = [HASHED_COSMOS_RECIPIENT_ADDRESS] // for testuser cosmos1pjtgu0vau2m52nrykdpztrt887aykue0hq7dfh, enter "0x636f736d6f7331706a74677530766175326d35326e72796b64707a74727438383761796b756530687137646668"
  _token = [DEPLOYED_TOKEN_ADDRESS] //erc20 not currently supported, enter "0x0000000000000000000000000000000000000000" for ethereum
  _amount = [WEI_AMOUNT]
```

Reply:
ah right, missed the _ thing. will fix in next push